packages/ses/src/scope-handler.js

```diff
@@ -65,6 +65,10 @@ export const createScopeHandler = (
         return undefined;
       }
 
+      if (prop === 'mellon') {
```

Comment:
What is `mellon`?

Reply:
https://www.youtube.com/watch?v=de19gIYgk-o

Reply:
This was clearly a reference aimed directly at my hobbies.